docs/install.md

```diff
@@ -20,9 +20,9 @@ bin/setup_conda [-n name] [-s spec]
 bin/setup_conda.bat
 ```
 
-These setup scripts perform full installation including Conda if not already present. Specifically, it will install:
+These scripts perform a full installation of Conda if not already present. Specifically, it will install:
 
-- [Miniconda](https://conda.io/miniconda.html), a light version of the Anaconda package and environment manager for Python, if not already present based on the `conda` command
+- [Miniconda](https://conda.io/miniconda.html), a package manager for the Anaconda Python Distribution, if not already present based on the `conda` command
```

Comment:
Nice catch. I just further tweaked the Miniconda description slightly based on its [website description](https://docs.conda.io/en/latest/miniconda.html).

```suggestion
- [Miniconda](https://conda.io/miniconda.html), a free minimal installer for the Anaconda Python Distribution, if not already present based on the `conda` command
```